server/src/modules/enquiry/enquiry.controller.ts

```diff
@@ -0,0 +1,61 @@
+import { createLogger } from '../../bootstrap/logging'
+import { ControllerHandler } from '../../types/response-handler'
+import { MailService } from '../mail/mail.service'
+import { Enquiry } from '../../types/mail-type'
+import { AgencyService } from '../agency/agency.service'
+
+const logger = createLogger(module)
+
+export class EnquiryController {
+  constructor(
+    private mailService: Public<MailService>,
+    private agencyService: Public<AgencyService>,
+  ) {}
+
+  postEnquiry: ControllerHandler<
+    unknown,
+    unknown,
+    {
+      agencyID: Array<string>
```

Comment:
```suggestion
      agencyId: Array<string>
```
and elsewhere, to be in line with conventions that `id` is stylised as lowercase in this codebase